scripts/gke/deployment_manager_configs/cluster.jinja

```diff
@@ -111,6 +111,7 @@ resources:
           machineType: n1-standard-8
           serviceAccount: {{ KF_VM_SA_NAME }}@{{ env['project'] }}.iam.gserviceaccount.com
           oauthScopes: {{ VM_OAUTH_SCOPES }}
+          minCpuPlatform: 'Intel Haswell'
```

Comment:
Can we add a comment explaining why we require this?

Reply:
Done